pepper-apis/src/main/java/org/broadinstitute/ddp/service/PdfGenerationService.java

```diff
@@ -845,6 +859,25 @@ private void substitutePicklist(Answer answer, PdfFormField field) {
         }
     }
 
+    private void substitutePicklist(PicklistAnswer answer, PdfAcroForm form, Map<String, String> mapping) throws IOException {
+        List<String> selectedOptions = new ArrayList<>();
+        if (answer != null) {
+            for (SelectedPicklistOption option : answer.getValue()) {
+                selectedOptions.add(option.getStableId());
+            }
+        }
+
+        for (String fieldName : mapping.keySet()) {
+            PdfFormField field = form.getField(fieldName);
+            if (field == null) {
+                LOG.warn(String.format("Could not find PDFFormField field with name: %s", fieldName));
```

Comment:
Should this be an error instead? Maybe we should pass in `List<String> errors` and append to it?

Reply:
Sure